Prace_domowe/Praca_domowa1/Grupa3/Jan_Smolen/Jan_Smolen_PD1.ipynb

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
<p>To samo co wyżej</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>